When react-i18next is updated to 12.0.0, we get npm ERR! ERESOLVE could not resolve on npm install

When we update react-i18next to latest version 12.0.0, we got the following error on `npm install`: 
```bash
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR! 
npm ERR! While resolving: storybook-react-i18next@1.1.2
npm ERR! Found: react-i18next@12.0.0
npm ERR! node_modules/react-i18next
npm ERR!   react-i18next@"12.0.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer react-i18next@"^11.17.1" from storybook-react-i18next@1.1.2
npm ERR! node_modules/storybook-react-i18next
npm ERR!   dev storybook-react-i18next@"1.1.2" from the root project
npm ERR! 
npm ERR! Conflicting peer dependency: react-i18next@11.18.6
npm ERR! node_modules/react-i18next
npm ERR!   peer react-i18next@"^11.17.1" from storybook-react-i18next@1.1.2
npm ERR!   node_modules/storybook-react-i18next
npm ERR!     dev storybook-react-i18next@"1.1.2" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

➞  **storybook-react-i18next** version 1.1.2

➞  npm -v 
8.15.0

➞  node -v 
v16.17.0

--- a/package.json
+++ b/package.json
@@ -69,7 +69,7 @@
     "prop-types": "^15.8.1",
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
-    "react-i18next": "^11.17.1",
+    "react-i18next": "^12.0.0",
     "rimraf": "^3.0.2",
     "typescript": "^4.7.3",
     "zx": "1.14.1"
@@ -85,7 +85,7 @@
     "i18next-http-backend": "^1.4.0",
     "react": "^16.8.0 || ^17.0.0 || ^18.0.0",
     "react-dom": "^16.8.0 || ^17.0.0 || ^18.0.0",
-    "react-i18next": "^11.17.1"
+    "react-i18next": "^11.17.1 || ^12.0.0"
   },
   "peerDependenciesMeta": {
     "react": {

